README.md

```diff
@@ -164,8 +173,7 @@ TODO: It would be great to document these variables further, and figure out whic
 
 ### Email
 
-This app sends emails via AWS: TODO more info here (we also have env variables set for Mandrill, are these still needed?)
-
+This app sends emails via the Amazon SES service. Look for the `config.action_mailer.delivery_method` in [production.rb](config/environments/production.rb) for the precise configuration details.
```

Comment:
Might be helpful to add: "If you need more information about our AWS setup, board members have console access."

Reply:
Done.